app/src/main/res/values/strings.xml

```diff
@@ -18,6 +18,7 @@
     <string name="where_are_you">Where are you?</string>
     <string name="search">Search</string>
     <string name="next_station">Next station: %1$s</string>
+    <string name="from_station">dep %1$s</string>
```

Comment:
Can we have the text as "from ..."?

So that it's "Fürstenried West (U), from Bahnhof, Gauting"

Reply:
imo Träwelldroid does not need to be an exact traewelling.de clone
I think, "from" or "dep. from" is more understandable